### ISSUE TITLE
Cannot restart all FPM servers using each

### ISSUE THREAD

**Issue by morozov** [phpbrew/phpbrew#558]
I'd like to be able to restart all FPM servers with a single command, however the `fpm` command is not recognized in the internal command call:

```
↪  phpbrew each -y fpm restart
Running `fpm` for php-5.6.12:
Did you mean 'app'? [Y/n]
```

It looks like currently you cannot pass multiple arguments to `each`. In `CommandBase::executeWrapper()`, `$args` is an array containing both arguments (`['fpm', 'restart']`), but `EachCommand::execute($command)` accepts only one of them and ignores the rest. Looks like it should be variadic.

Passing `"fpm restart"` as a single argument seems to do the trick, and internal `PhpBrew::run($args)` is called with two arguments, but the command is not recognized anyways since it turns out to be implemented on the shell level, not in PHP:

```
↪  phpbrew each -y "fpm restart"
Running `fpm restart` for php-5.6.12:
Did you mean 'app'? [Y/n]
```

@c9s, @marcioAlmada is anything above by design?

**Comment by c9s** [comment]
The fpm command was implemented in shell, and the each command looks like implemented in php

**Comment by morozov** [comment]
Fixed by #888.
